utils/obsproc/IcecAmsr2Ioda.h

```diff
@@ -66,18 +69,31 @@ namespace gdasapp {
       iodaVars.referenceDate_ = "seconds since 1970-01-01T00:00:00Z";
 
       size_t index = 0;
-      std::tm timeinfo = {};
       for (int i = 0; i < ntimes; i += dimTimeSize) {
-        timeinfo.tm_year = oneTmpdateTimeVal[i] - 1900;  // Year since 1900
-        timeinfo.tm_mon = oneTmpdateTimeVal[i + 1] - 1;  // 0-based; 8 represents Sep
-        timeinfo.tm_mday = oneTmpdateTimeVal[i + 2];
-        timeinfo.tm_hour = oneTmpdateTimeVal[i + 3];
-        timeinfo.tm_min = oneTmpdateTimeVal[i + 4];
-        timeinfo.tm_sec = oneTmpdateTimeVal[i + 5];
-
-        // Calculate and store the seconds since the Unix epoch
-        time_t epochtime = std::mktime(&timeinfo);
-        iodaVars.datetime_(index) = static_cast<int64_t>(epochtime);
+        int year = oneTmpdateTimeVal[i];
+        int month = oneTmpdateTimeVal[i+1];
+        int day = oneTmpdateTimeVal[i+2];
+        int hour = oneTmpdateTimeVal[i+3];
+        int minute =  oneTmpdateTimeVal[i+4];
+        int second = static_cast<int>(oneTmpdateTimeVal[i+5]);
+
+        // Bypassing validYYYYMMDD function within dateToJulian

```

Comment:
I'm not sure the comment fits what's being done below.

Reply:
> I'm not sure the comment fits what's being done below.

Will write down more clearly-